edx-platform/pearson-vue-theme/lms/templates/header/navbar-not-authenticated.html

```diff
@@ -15,7 +15,7 @@
 <%
   mktg_site_enabled = static.get_value('ENABLE_MKTG_SITE', settings.FEATURES.get('ENABLE_MKTG_SITE', False))
   course_link_enable = static.get_value('COURSE_LINK_ENABLE', settings.FEATURES.get('COURSE_LINK_ENABLE', False))
-  allows_login = not settings.FEATURES['DISABLE_LOGIN_BUTTON'] and not combined_login_and_register
+  allows_login = configuration_helpers.get_value('ENABLE_LOGIN_BUTTON', True)
```

Comment:
Let's make the "DISABLE_LOGIN_BUTTON" setting site aware instead of creating a new setting. @luismorenolopera

Reply:
Done @Squirrel18